api/src/opentrons/protocol_engine/errors/exceptions.py

```diff
@@ -82,3 +86,11 @@ class ProtocolEngineStoppedError(ProtocolEngineError):
 
 class WellOriginNotAllowedError(ProtocolEngineError):
     """An error raised when using a disallowed origin in a relative well location."""
+
+
+class NoAttachedModuleFound(ProtocolEngineError):
```

Comment:
For parity with pipette loading

```suggestion
class ModuleNotAttachedError(ProtocolEngineError):
```